### PR TITLE
rename invariant to warning

### PR DESCRIPTION
**Pull request by adnasa**
#### Summary

- renames `invariant` tool to `warning`
- update imports and usage

### CODE REVIEW

File: packages/utils/src/warning.ts [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+const warning = (condition: unknown, message?: string): asserts condition => {
+  if (isProduction) {
     return;
   }
-  // GIVEN product, suppress the message.
-  if (isProduction) {
+
+  if (condition) {
     return;
   }
```
Comment (mohib0306) [inline packages/utils/src/warning.ts:15]: can we put these into single OR statement?

**Review by mohib0306 — APPROVED** [review]
Looks good

File: packages/utils/src/warning.ts (hunk 1 above)
Comment (adnasa) [reply packages/utils/src/warning.ts:15]: I was tempted to do it, but I like how this is structured actually #oldschool  😅

**Review by tdeekens — APPROVED** [review]
💯

**Pull request merged**